src/app.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable indent */
+import express from 'express';
+
+import bodyParser from 'body-parser';
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(express.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+const port = process.env.PORT || 5000;
+
+app.get('/', (req, res) => res.status(200).send({
+        status: 200,
+        message: 'Hey there'
+    }));
+
+app.listen(port, () => {
+  console.log(`Server has started on port ${port}!!!`);
+});
```

Comment:
@eustacheAki worked on this feature you can delete this file

Reply:
DOne